boot/bootutil/include/bootutil/image.h

```diff
@@ -85,7 +85,6 @@ struct flash_area;
 #define IMAGE_TLV_PUBKEY            0x02   /* public key */
 #define IMAGE_TLV_SHA256            0x10   /* SHA256 of image hdr and body */
 #define IMAGE_TLV_RSA2048_PSS       0x20   /* RSA2048 of hash output */
-#define IMAGE_TLV_ECDSA224          0x21   /* ECDSA of hash output */
```

Comment:
I think it's better to leave the TLV definition here, so it's not reused, just change the comment to state it's not supported anymore.

Reply:
Hi Fabio,
I have no problem with marking it as not supported, I just would like to understand the reasoning behind it.
I don't see how reusing the value 0x21 for a different TLV type would cause any problems.
Thanks.

Reply:
Why would you break compatibility? `newt` for example is able to create images using p224. If someone is actually using that curve, why would I parse this TLV as some other type of data instead of just failing? How can you know who else might be using other tooling with MCUboot without publishing it to the world?

Reply:
Thanks for the answer, it gave me a new perspective.
I understand, but there won't be any compatibility anyway - if we remove the P224 support.
However, I completely agree with this: it is much better to fail during TLV parsing with a missing TLV error than fail because of its unexpected content.

Reply:
I've put it back, and marked it with not supported anymore.